tests/v2/validation/publicapi/clusterroletemplatebindings/crtb_create_steve_test.go

```diff
@@ -0,0 +1,292 @@
+//go:build (validation || infra.any || cluster.any || sanity) && !stress && !extended
+
+package clusterroletemplatebindings
+
+import (
+	"encoding/json"
+	"testing"
+
+	v3 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
+	management "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+	namegen "github.com/rancher/rancher/tests/framework/pkg/namegenerator"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	"github.com/rancher/rancher/tests/framework/extensions/clusters"
+	"github.com/rancher/rancher/tests/framework/extensions/users"
+	"github.com/rancher/rancher/tests/framework/pkg/session"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type CRTBGenTestSuite struct {
+	suite.Suite
+	client            *rancher.Client
+	crtbName          string
+	roleTemplateName  string
+	userPrincipalName string
+	session           *session.Session
+	clusterName       string
+	clusterID         string
+
+	newUser              *management.User
+	crtbJsonTemplateData []byte
+}
+
+const (
```

Comment:
can we have a common helper file for the publicAPI and move all the constants there? so we can reuse them for other tests as well?